yaml/csi-driver/edge/hpe-csi-k8s-1.21.yaml

```diff
@@ -739,12 +739,6 @@ rules:
   - apiGroups: ["storage.hpe.com"]
     resources: ["hpereplicationdeviceinfos"]
     verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
-  - apiGroups: ["storage.hpe.com"]
-    resources: ["hpevolumegroupinfos"]
-    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
-  - apiGroups: ["storage.hpe.com"]
-    resources: ["hpesnapshotgroupinfos"]
-    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]    
```

Comment:
What is the purpose of this removal?

Reply:
Interesting. I based this yaml off 1.20 and it is missing there as well. Let me fix both